department-of-veterans-affairs/va.gov-cms
# VACMS-9304: Show am/pm on start and end in recurring events UI

… to suppress them.

## Description

Closes #9304.

## Screenshots
<img width="800" alt="Screen Shot 2022-06-16 at 3 32 54 PM" src="https://user-images.githubusercontent.com/221539/174329455-b1f56943-6956-40ed-a93e-2c6abc47233a.png">

![Screen Shot 2022-06-17 at 9 10 37 AM](https://user-images.githubusercontent.com/221539/174336803-fda7e774-c11f-4422-9c6b-617468c9e8c0.png)

## QA steps

As a content creator
1. Create a new Event for any Facility. Ensure that the event is a recurring event, and the date/time for start and end dates occurs on the same day.
   - [x] Validate that when viewing the new Event, the event times contain am/pm for both start and end times.
2. Then edit the node, and click the 'Edit event series' link.
   - [x] Validate that am/pm is visible for both start and end times.

### Definition of Done

### Select Team for PR review

- [] `⭐️ Sitewide CMS`
- [x ] `⭐️ Facilities`


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
